kazaam_int_test.go

```diff
@@ -149,6 +149,32 @@ func TestKazaamCoalesceTransformAndShift(t *testing.T) {
 	}
 }
 
+func TestKazaamShiftTransformWithTimestamp(t *testing.T) {
+	spec := `[{
+		"operation": "shift",
+		"spec": {"newTimestamp":"oldTimestamp","oldTimestamp":"oldTimestamp"}
+	}, {
+		"operation": "timestamp",
+		"spec": {"newTimestamp":{"inputFormat":"Mon Jan _2 15:04:05 -0700 2006","outputFormat":"2006-01-02T15:04:05-0700"}}
+	}]`
+
+	// for some reason, keys are inserted in different order on different runs locally and in CI
+	// so without the alt we get sporadic failures.
+	jsonIn := `{"oldTimestamp":"Fri Jul 21 08:15:27 +0000 2017"}`
+	jsonOut := `{"oldTimestamp":"Fri Jul 21 08:15:27 +0000 2017","newTimestamp":"2017-07-21T08:15:27+0000"}`
+	altJsonOut := `{"newTimestamp":"2017-07-21T08:15:27+0000","oldTimestamp":"Fri Jul 21 08:15:27 +0000 2017"}`
```

Comment:
Yeah something along those lines. A number of other tests need a similar treatment, I'll open a ticket.